src/pages/_index/_components/what-is-wasmcloud/index.tsx

```diff
@@ -0,0 +1,188 @@
+import React from 'react';
+import { Section } from '@site/src/pages/_components/section';
+import { SectionTag } from '@site/src/pages/_components/section-tag';
+import { SectionHeading } from '@site/src/pages/_components/section-heading';
+import {
+  Switcher,
+  SwitcherButton,
+  SwitcherContent,
+  SwitcherList,
+} from '../what-is-wasmcloud/switcher';
+import { SectionSubheading } from '@site/src/pages/_components/section-subheading';
+import styles from './what-is-wasmcloud.module.css';
+
+type Props = {};
+
+const INTRO_CONTENT: {
+  tag: React.ReactNode;
+  heading: React.ReactNode;
+  description: React.ReactNode;
+} = {
+  tag: 'What is wasmCloud?',
+  heading: 'Fast-track the development, deployment, and management of distributed applications',
+  description: (
+    <p>
+      wasmCloud is an open source project from the Cloud Native Computing Foundation (CNCF) that
+      enables teams to build polyglot applications composed of reusable Wasm components and run
+      them—resiliently and efficiently—across any cloud, Kubernetes, datacenter, or edge.
+    </p>
+  ),
+};
+
+const SWITCHER_CONTENT: Array<{
+  id: string;
+  title: React.ReactNode;
+  image: string;
+  features: Array<{
+    title: React.ReactNode;
+    description: React.ReactNode;
+    link?: string;
+    linkText?: React.ReactNode;
+  }>;
+}> = [
+  {
+    id: 'build',
+    image: '/img/pages/home/what-is-wasmcloud/build.svg',
+    title: 'Build',
+    features: [
+      {
+        title: 'Applications Faster',
+        description: (
+          <>
+            Leveraging reusable, polyglot, Wasm components on a reliable, distributed platform.
+          </>
+        ),
+      },
+      {
+        title: 'Centrally Maintainable Apps',
+        description: (
+          <>
+            Reusable, version-controlled, components empower platform teams to maintain 1,000's of diverse apps centrally.
+          </>
+        ),
+      },
+      {
+        title: 'Compatible With Existing Platforms & Stacks',
+        description: (
+          <>
+            Common integrations make the wasmCloud ecosystem compatible with today's platforms - Argo, Jenkins, Actions, 
+            Chainguard, Backstage, K8s...
```

Comment:
```suggestion
            Chainguard, Backstage, K8s, and more.
```

Feels a little stronger here instead of the drop-off